Tests/js/views/ez-rawcontentview.html

```diff
@@ -19,6 +20,32 @@
 
 <script type="text/x-handlebars-template" id="fieldview-ez-template">fieldview</script>
 
+<script type="text/x-handlebars-template" id="languageswitcherview-ez-template">
+    <div class="ez-selected-language">
```

Comment:
usually in test we put a simplified and minimal version of the template. In this case, this would also avoid having to declare the ez-getflagcodehelper dependency.